deploy_from_github_zip.bash

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+[ $# -eq 0 -o $# -gt 3 ] && {
+    echo "Usage: deploy_from_github_zip.bash <new project name> [GitHub-user/repository-name] [branch]"
+    exit 1
+}
+
+project=$1
```

Comment:
Giving the user complete control of the project name is liberating, but this is different from the original `start_paper` concept. There, the name of the Note followed a particular format, that I was guessing the DESC pub system might adopt. This is just one model though - we can instead require the Pub System to cope with any paper name we as users choose. BTW we shoudl call this variable "paper" instead of "project" to avoid confusion with the DESC term "project" which does not equal "paper".